README.md

```diff
@@ -11,10 +11,89 @@ This project serves as a compilation of examples that highlight the fundamental
 
 Here you'll find examples of the following features:
 
-- [Section Override V2](https://developers.vtex.com/docs/guides/faststore/overrides-overview)
-- [API Extension](https://developers.vtex.com/docs/guides/faststore/api-extensions-overview)
+- 👉 [Section Override V2](#1-section-override-v2)
+  - [Adding icon options to the Alert component](#adding-icon-options-to-the-alert-component)
+  - [Adding an image to your Alert component](#adding-an-image-to-your-alert-component)
+- 👉 [API Extensions](#2-api-extensions)
+  - [Adding Installment Information in the Product Details Page](#adding-installment-information-in-the-product-details-page)
+  - [Adding a contact form to a landing page](#adding-a-contact-form-to-a-landing-page)
 
-### Naming Conventions
+## 👩‍🏫 How to use this repository
+
+The repository contains folders/files with code [examples](#-examples) for the features mentioned above.
+
+**Follow Along with Documentation**:
+As you run the examples, refer to the documentation to understand the concepts and implementation details of each part of the code. The documentation will guide you through the expected output and any configurations needed.
+
+**Experiment and Modify**: Feel free to experiment with the code examples by making modifications. This will help you gain a deeper understanding of how each feature works.
+
+Clone the repository in your local machine and start exploring:
+
+1. Clone the repository to your local machine:
+
+```
+git clone https://github.com/vtex-sites/playground.store.git
+```
+
+2. Navigate to the repository directory:
```

Comment:
Before this step, should we recommend the person create a new branch?

Reply:
I don't think its necessary, because they probably won't send anything to this repo 🤔